src/components/FormInput.tsx

```diff
@@ -1,29 +1,52 @@
 import React, { useState } from 'react';
+import TodoListProps from '../TypescriptProps';
 
 interface FormInputProp {
-    addTask: any;
+    addTask: (userInput: string ) => void;
+    listOfItems: TodoListProps[];
 }
 
-const FormInput: React.FC<FormInputProp> = ({ addTask }) => {
-
+const FormInput: React.FC<FormInputProp> = ({ addTask, listOfItems }) => {
     const [ userInput, setUserInput ] = useState('');
-
+    const [ inputError, setInputError ] = useState('');
+    let listCheck = listOfItems.map(listDes => listDes.description);
+    const errorCheck = () => {
```

Comment:
good work on the error checks! could add another one - what if the user enters too many characters?